worker/src/index.ts

```diff
@@ -11,7 +11,6 @@ const main = (async () => {
 
   const consumer = await buildConsumer({
     connection: jetstreamConnection,
-    stream: 'DebeziumStream',
```

Comment:
Can we please return this value passed as an argument? Anything outside the `core` is just an example, and devs might customize and use it differently.

Reply:
This is reverted